composable_kernel/include/tensor_operation/element_wise_operation.hpp

```diff
@@ -12,6 +12,31 @@ struct PassThrough
     __host__ __device__ void operator()(half_t& y, const half_t& x) const { y = x; }
 };
 
+struct Add
+{
+    template <typename T1, typename T2, typename T3>
```

Comment:
Let's not use templated type, but real type here

Reply:
Fix
https://github.com/ROCmSoftwarePlatform/composable_kernel/commit/9bafd938f7cf09a809c0183074cf30f8664cd93b#diff-990561d35152da39a727f324616ba178793b8b1e20ecd17194b48988282cbd66R15-R30